docs/source/user_guide/troubleshooting.rst

```diff
@@ -0,0 +1,60 @@
+.. _user_guide_troubleshooting:
+
+===============
+Troubleshooting
+===============
+In this page, users can find ideas to resolve the most common issues encountered with pydpf-core.
+Ways to improve users codes will also be exposed.
```

Comment:
I'd write this as:
This section also includes suggestions for how to improve your scripts that use ``pydpf-core``.